File: terraform/src/sendMessageToSQS.js

const AWS = require('aws-sdk');
const SQS = new AWS.SQS();
/* 宛先QueueのURL */
const QUEUE_URL = 'hogehoge.sqs';

exports.handler = function(event, context) {

    // SendMessage
    var params = {
        MessageBody: QUEUE_URL
    };

    console.log(params);
};

File: terraform/src/send_message_to_sqs.py
import boto3
import json

def lambda_handler(event: dict, context):
    client = boto3.client("sqs")
    response = client.send_message(
        QueueUrl="https://sqs.ap-northeast-1.amazonaws.com/086656038367/score-evaluation-queue",
        MessageBody=json.dumps(event)
    )
    return response